Source/GTMSessionUploadFetcher.m

```diff
@@ -374,6 +382,27 @@ + (NSArray *)uploadFetchersForBackgroundSessions {
   return uploadFetchers;
 }
 
+- (dispatch_queue_t)chunkWorkQueue {
+  @synchronized(self) {
+    if (!_chunkWorkQueue) {
+      dispatch_queue_t targetQueue = dispatch_get_global_queue(QOS_CLASS_DEFAULT, 0);
+#if TARGET_OS_IOS && (__IPHONE_OS_VERSION_MIN_REQUIRED < __IPHONE_10_0)
```

Comment:
I sorta feel like we have this pattern for queue creation a bunch, does it make sense to make a single (inline?) helper for it to reduce the boilerplate until the min iOS version is moved up? (maybe in a follow up PR)

Reply:
We've only got one other location at the moment. I was thinking about that, but thought I'd maybe hold off until making a later change to allow setting a worker target queue on the service/fetcher, to allow app subsystems that want to follow Apple's GCD recommendations on a queue hierarchy to set the target for the network activity.